seacrowd/sea_datasets/ntrex_128/ntrex_128.py

```diff
@@ -0,0 +1,169 @@
+      Patra, Barun  and
+      Neubig, Graham  and
+      Choudhury, Monojit  and
+      Dandapat, Sandipan  and
+      Sitaram, Sunayana  and
+      Chaudhary, Vishrav",
+    booktitle = "Proceedings of the First Workshop on Scaling Up Multilingual Evaluation",
+    month = nov,
+    year = "2022",
+    address = "Online",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2022.sumeval-1.4",
+    pages = "21--24",
+}
+"""
+
+_DATASETNAME = "ntrex_128"
+
+_DESCRIPTION = """\
+NTREX-128, a data set for machine translation (MT) evaluation, includes 123 documents \
+(1,997 sentences, 42k words) translated from English into 128 target languages. \
+9 languages are natively spoken in Southeast Asia, i.e., Burmese, Filipino, \
+Hmong, Indonesian, Khmer, Lao, Malay, Thai, and Vietnamese.
+"""
+
+_HOMEPAGE = "https://github.com/MicrosoftTranslator/NTREX"
+
+_LANGUAGES = ["mya", "fil", "ind", "khm", "lao", "zlm", "tha", "vie", "hmv"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+_LICENSE = Licenses.CC_BY_SA_4_0.value  # example: Licenses.MIT.value, Licenses.CC_BY_NC_SA_4_0.value, Licenses.UNLICENSE.value, Licenses.UNKNOWN.value
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: "https://raw.githubusercontent.com/MicrosoftTranslator/NTREX/main/NTREX-128/newstest2019-ref.{lang}.txt",
+}
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+
+_SOURCE_VERSION = "11.24.2022"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class Ntrex128Dataset(datasets.GeneratorBasedBuilder):
+    """NTREX-128, a data set for machine translation (MT) evaluation, includes 123 documents \
+    (1,997 sentences, 42k words) translated from English into 128 target languages. \
+    9 languages are natively spoken in Southeast Asia, i.e., Burmese, Filipino, \
+    Hmong, Indonesian, Khmer, Lao, Malay, Thai, and Vietnamese."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_{subset1}_{subset2}_source",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description=f"{_DATASETNAME} {subset1}2{subset2} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}_{subset1}_{subset2}",
+        )
+        for subset2 in _LANGUAGES
+        for subset1 in _LANGUAGES
+        if subset1 != subset2
+    ] + [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_{subset1}_{subset2}_seacrowd_t2t",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description=f"{_DATASETNAME} {subset1}2{subset2} SEACrowd schema",
+            schema="seacrowd_t2t",
+            subset_id=f"{_DATASETNAME}_{subset1}_{subset2}",
+        )
+        for subset2 in _LANGUAGES
+        for subset1 in _LANGUAGES
```

Comment:
Hi @holylovenia, there are 2 things as follow:
1. Regarding this, should we add macro languages such as `hmn` and `msa` into the list or we keep the current standard?
2. NTREX is a parallel dataset for 128 languages. Should we take all subsets with one of the languages are from SEA? In this case, there will be a modification required when defining the subsets.

Reply:
@SamuelCahyawijaya Hi! As I understood I needed to change these language codes during downloading dataset from github. But for schema name it should stay zlm and hmv, right? I have done these changes, if something wrong just let me know.

Regarding 2nd comment, I think that it would be great to add other languages, where for translation it would be combination of one SEA language and one non SEA language.

Reply:
@MJonibek : 
Yes, we can keep it that way for the 1st comment. Thanks for the update!
For the 2nd comment, I have asked @holylovenia, and we will add all the language pair with **at least one SEA languages**. 

Would you mind to revise it accordingly? Thank you!

Reply:
@SamuelCahyawijaya I was going through language codes, can you please help: which language codes I should use for these languages:

aze-Latn: Azerbaijani (Latin)
ckb-Arab: Central Kurdish (Sorani)
eng-GB: English (British), eng-IN: English (India), eng-US: English (US)
fra: French, fra-CA: French (Canada)
mya: Myanmar 
por: Portuguese, por-BR: Portuguese (Brazil)
shi: Shilha
sna-Latn: Shona (Latin)
snd-Arab: Sindhi (Arabic)
spa: Spanish, spa-MX: Spanish (Mexico)
srp-Cyrl: Serbian (Cyrillic), srp-Latn: Serbian (Latin)
tgk-Cyrl: Tajik (Cyrillic)
yue: Cantonese
zho-CN: Chinese (Simplified), zho-TW: Chinese (Traditional)

These are languages and their corresponding language codes in the ntrex 128 dataset

Reply:
For now I left it as it is

Reply:
Hi @MJonibek, sorry for the late reply.  I think we can use all of them, implementing all languages at the regional/dialectical level can be extremely useful. So, let's keep it in this way 👍🏻